libnavigator/build.gradle

```diff
@@ -50,12 +50,7 @@ dependencies {
     implementation dependenciesList.mapboxAndroidCommon
 
     // Network
-    implementation(dependenciesList.okhttp) {
-        force = true
-    }
-    implementation(dependenciesList.okhttpInterceptor) {
-        force = true
-    }
+    implementation dependenciesList.okhttpInterceptor
```

Comment:
Is this dep needed? Looks like it's unused.

Reply:
seems redundant, removed 👍